src/main/java/org/example/constant/UrlPath.java

```diff
@@ -4,7 +4,7 @@
 import java.util.Map;
 
 public enum UrlPath {
-    HTML("/html"), JSON("/json"), HELLO("/hello");
+    HTML("/html"), JSON("/json"), HELLO("/hello"), CREATION_FORM("/form");
```

Comment:
Refactor the API. Make it more usable and meaningful.

Reply:
tried

Reply:
API still not meaningful

Reply:
FIXED

Reply:
API still not meaningful

Reply:
changed